packages/parcel/src/parcel-process.ts

```diff
@@ -0,0 +1,86 @@
+import * as Path from 'path';
+import { ChildProcess, fork as forkProcess } from 'child_process';
+
+import { monitor, Operation } from 'effection'
+import { Mailbox, once, suspend } from '@bigtest/effection';
+
+interface ParcelOptions  {
+  buildDir: string;
+  srcPath: string;
+  stdio?: 'pipe' | 'inherit';
+  execPath?: 'ts-node' | undefined;
+}
+
+export class ParcelProcess {
+  mailbox: Mailbox = new Mailbox();
+
+  static *create(options: ParcelOptions): Operation {
+    let stdioMode = options.stdio || 'pipe';
+    let parcelProcess = new ParcelProcess();
+
+    let runParcel = Path.join(__dirname, 'parcel-run');
+    let child: ChildProcess = forkProcess(
+      runParcel,
+      ['--out-dir', options.buildDir, '--out-file', 'manifest.js', '--global', '__bigtestManifest', options.srcPath],
+      {
+        execPath: options.execPath || undefined,
```

Comment:
I think we probably don't need this line?

Reply:
This was for the case where we are running inside the tests which Stilll happen in ts-node. I couldn’t get it to work without the exec path. Maybe we could run against the preconoiled code in the tests.... in any event, The `|| undefined` is a bit silly

Reply:
This was for the case where we are running inside the tests which Stilll happen in ts-node. I couldn’t get it to work without the exec path. Maybe we could run against the preconoiled code in the tests.... in any event, The `|| undefined` is a bit silly

Reply:
Just realized we're using this for the tests :)